### ISSUE TITLE
"List events" on homepage

### ISSUE THREAD

**Issue by jeffmcq1616** [spring-giant/mgt656-spring-2015-project#8]
Each event must have details. Events that are already completed should not show up.

**Comment by jeffmcq1616** [comment]
A an unordered list of events, including their details

```
Grading Note: each event is in an li element with class="event" and id="event-{id}", where {id} indicates the numerical id of the event. E.g. your event list may look something like the following:

  <ul>
    <li class="event" id="event-1">...</li>
    <li class="event" id="event-2">...</li>
  </ul>

Each list element displays pertinent information about the event: title, date, and how many people are RSVPd etc. It should include the default events that come with the project. E.g. in the above example those details should be where you see "...".
Every event li element should have an HTML5 time tag that has a datetime attribute equal to the time in ISO 8601 format. The text of the tag should be the datetime in natural language, including the time. For example, it may look like

  <time datetime="2014-11-12 09:30:0.0+07:00">
    Wednesday, Nov 12 at 9:30AM
  </time>
```

**Comment by meghaghildyal** [comment]
Hi Jeff, I added some more to your event page to make it more substantial. it seems teams are spending between 2-6 hours for each sprint. This means that at least 2 hours for each story. I think this is because people are adding in CSS (Style) elements to their work as well. What I have added to your test this time is this:

The title for each event is a clickable link to the "detail page" for that event, which should be at /events/, where id refers to the id of the event, e.g. /events/0 would take me to the event with id zero.

The test passes when I do mocha -g link. 

The conclusion from this is that each one of us should now take up a user story for each sprint and assign 'time' to it that is anywhere between 2-6 hours. We can discuss this during the sprint review meetings.

-Megha
